### PR TITLE
Seedlet: Add additional gutenberg-related theme tags

### PR DESCRIPTION
**Pull request by kjellr**
This PR adds a few extra [theme tags](https://make.wordpress.org/themes/handbook/review/required/theme-tags/) for Seedlet: 

- `block-styles` (because it includes block styles)
- `block-patterns` (because it includes block patterns)
- `wide-blocks` (because it supports wide & full alignments)

### CODE REVIEW

File: seedlet/style-rtl.css [hunk 1]
```diff
@@ -5,13 +5,12 @@ Author: Automattic
 Author URI: https://automattic.com/
 Description: Seedlet is a free WordPress theme. A two-column layout and classically elegant typography creates a refined site that gives your works and images space to breathe - and shine. Seedlet was built to be the perfect partner to the block editor, and supports all the latest blocks. Writing, audio, illustrations, photography, video - use Seedlet to engage and direct visitors' eyes, without your theme getting in the way. And the responsive design shifts naturally between desktop and mobile devices. Seedlet is a great option for professionals and creatives looking for a sophisticated vibe. Whether you're looking to create a blog or a robust site promoting your business, do with simplicity, style, and Seedlet.
 Requires at least: 4.9.6
-Tested up to: 5.4.1
-Requires PHP: 7.3
-Version: 1.0.3-wpcom
+Tested up to: 5.5
+Version: 1.0.6-wpcom
```
Comment (kjellr) [inline seedlet/style-rtl.css:9]: It looks like these are left over from a previous commit that wasn't fully compiled?

File: seedlet/style-rtl.css [hunk 2]
```diff
@@ -1398,12 +1397,12 @@ object {
 	margin-bottom: var(--global--spacing-vertical);
 }
 
-.wp-block-cover .wp-block-cover__inner-container a,
-.wp-block-cover .wp-block-cover-image-text a,
-.wp-block-cover .wp-block-cover-text a,
-.wp-block-cover-image .wp-block-cover__inner-container a,
-.wp-block-cover-image .wp-block-cover-image-text a,
-.wp-block-cover-image .wp-block-cover-text a {
+.wp-block-cover .wp-block-cover__inner-container a:not(.has-text-color),
```
Comment (scruffian) [inline seedlet/style-rtl.css:1400]: where is this change coming from?

**Comment by scruffian** [comment]
I think we should also add `editor-styles`...

File: seedlet/style-rtl.css (hunk 2 above)
Comment (kjellr) [reply seedlet/style-rtl.css:1400]: That appears to be left over from this merged PR: https://github.com/Automattic/themes/pull/2417/

For some reason, the compile script doesn't always catch the RTL styles.

**Comment by kjellr** [comment]
@scruffian It already includes the `editor-style` tag (not `editor-styles`). That's the one noted in the [Theme Review guidelines](https://make.wordpress.org/themes/handbook/review/required/theme-tags/).

**Review by scruffian — APPROVED** [review]
🚢

**Pull request merged**